### PR TITLE
DDP-4244 configure verify governed user email

### PR DESCRIPTION
**Pull request by ssettipalli**
look at https://broadinstitute.atlassian.net/browse/DDP-4244
This PR depends on https://github.com/broadinstitute/ddp-study-server/pull/30 to be able to merge

### CODE REVIEW

**Comment by ssettipalli** [comment]
please run tests

File: study-builder/studies/osteo/study.conf [hunk 1]
```diff
@@ -1815,6 +1830,20 @@
...
+        "type": "INVITATION_CREATED"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
+        "emailTemplate": ${emails.reconsentEmailVerification},
+        "language": "en",
+        "pdfAttachments": []
+      },
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
```
Comment (yufengwng) [inline study-builder/studies/osteo/study.conf]: Email events are dispatch, so this needs to be `true`.

File: study-builder/studies/osteo/study.conf [hunk 2]
```diff
@@ -1815,6 +1830,20 @@
       "dispatchToHousekeeping": true,
       "order": 1
     },
+    {
+      "trigger": {
+        "type": "INVITATION_CREATED"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
```
Comment (yufengwng) [inline study-builder/studies/osteo/study.conf]: I made some slightly different design decisions. Based on that, this should be `INVITATION_EMAIL` type instead. See PR #30.

**Review by yufengwng — COMMENTED** [review]
Looks good overall!

**Comment by ssettipalli** [comment]
please run tests

File: study-builder/studies/osteo/study.conf [hunk 3]
```diff
@@ -1815,6 +1830,20 @@
       "dispatchToHousekeeping": true,
       "order": 1
     },
+    {
+      "trigger": {
+        "type": "INVITATION_EMAIL"
```
Comment (yufengwng) [inline study-builder/studies/osteo/study.conf]: Sorry for not being clear, what we should do here is actually:

* Use trigger type `INVITATION_CREATED`
* Use action type `INVITATION_EMAIL`

This way, after the invitation is created by the event above, we will then send out an email with the invitation guid embedded. Does that make sense?

**Comment by ssettipalli** [comment]
please run tests

**Review by yufengwng — DISMISSED** [review]
LGTM

**Comment by ssettipalli** [comment]
please run tests

**Review by yufengwng — APPROVED** [review]

**Pull request merged**